README.md

```diff
@@ -25,6 +25,10 @@ The script will also look for merged pull requests that have the labels
 It will also search for pull requests that target release branches and remove
 any `backport/*` labels from them.
 
+It will also make sure any pull requests that have
+`## :warning: BREAKING :warning:` in their description have the `kind/breaking`
+label.
```

Comment:
Also the other way around?

Reply:
What do you mean?

Reply:
`It will make sure that any pull request with the `kind/breaking` label have
`## :warning: BREAKING :warning:` in their description.`

Reply:
With a check similar to `giteabot/lgtm`?

Reply:
Yes.

Reply:
Interesting. Another option would also be to remove `kind/breaking` if ``## :warning: BREAKING`` is not in the description automatically, like we do on PRs targeting release branches (we remove backport labels from them)

Reply:
Hmm. I don't think it's a good idea to remove a label that is intended as protection.
The backport labels have a functional purpose that doesn't apply under the given circumstances.
But a PR is not not-breaking given that the disclaimer is missing.